templates/operator.yaml.j2

```diff
@@ -95,6 +95,8 @@ spec:
                       type: integer
                 type:
                   type: string
+                onDelete:
```

Comment:
How about using `delete` as default here and remove `onDelete: delete` from all storage-classes?
Then we just need to disregard value for this key only for `external non-native storage`, thoughts?

PS: Don't forget update in helm as well :)

Reply:
Yes its default, I have added it has a reference. like onDelete = obj["spec"].get("onDelete", "delete")

Reply:
I'll update the docs once I complete this PR properly.

Reply:
> Yes its default, I have added it has a reference. like onDelete = obj["spec"].get("onDelete", "delete")

ack, I meant using default in crd itself like below, disregard if you mean the same:
``` yaml
onDelete:
  type: string
  default: delete
```

Reply:
cool will do it, I had meant the other one